numba_dpex/core/utils/kernel_launcher.py

```diff
@@ -151,11 +204,11 @@ def build_complex_arg(self, val, ty, arg_list, args_ty_list, arg_num):
         )
         arg_num += 1
 
-    def build_array_arg(
+    def _build_array_arg(  # pylint: disable=too-many-arguments
         self,
         array_val,
         array_data_model,
-        array_rank,
+        array_rank,  # pylint: disable=unused-argument
```

Comment:
Why not remove?

Reply:
I've thought it was pending TODO. Removed